app/models/admin.rb

```diff
@@ -0,0 +1,4 @@
+class Admin < ActiveRecord::Base
+  validates :username, presence: true
```

Comment:
I think uniqueness would be good. I will add that in.
This is the super admin... so an opportunity@work person.